anesthetic/weighted_pandas.py

```diff
@@ -184,10 +184,8 @@ def sem(self, skipna=True):
         """Weighted standard error of the mean."""
         return np.sqrt(self.var(skipna=skipna)/self.neff())
 
-    def quantile(self, q=0.5, numeric_only=True, interpolation='linear'):
+    def quantile(self, q=0.5, interpolation='linear'):
```

Comment:
The `Series.quantile()` signature doesn't actually include a `numeric_only` kwarg, so I've removed it here.

Reply:
Fair.